package.json

```diff
@@ -1,23 +1,22 @@
 {
   "name": "wallet-contracts",
-  "version": "1.0.1",
+  "version": "2.0.0",
   "private": false,
   "license": "none",
   "scripts": {
     "prepare": "npm run build",
-    "build": "buidler compile",
-    "postbuild": "yarn gen:contract-typings && tsc -b",
+    "build": "hardhat compile",
+    "postbuild": "yarn gen:contract-typings",
     "clean": "rimraf artifacts && rimraf typings/contracts && rimraf cache",
     "pretest": "yarn build",
-    "test": "buidler test",
-    "quicktest": "buidler test",
-    "test:truffle": "tsc -b && truffle test --network ganache build/*.spec.js",
+    "test": "hardhat test",
+    "quicktest": "hardhat test",
     "precoverage": "yarn build",
-    "coverage": "COVERAGE=true NET_ID=1 buidler coverage --network coverage",
+    "coverage": "COVERAGE=true NET_ID=1 hardhat coverage --network coverage",
     "benchmark": "BENCHMARK=true yarn test",
     "gen:contract-typings": "rimraf typings/contracts/* && yarn gen:contract-typings:ethers4 && yarn gen:contract-typings:ethers5",
-    "gen:contract-typings:ethers4": "typechain --target ethers-v4 --outDir typings/contracts/ethers-v4 './artifacts/*.json'",
-    "gen:contract-typings:ethers5": "typechain --target ethers-v5 --outDir typings/contracts/ethers-v5 './artifacts/*.json'",
+    "gen:contract-typings:ethers4": "typechain --target ethers-v4 --outDir typings/contracts/ethers-v4 './artifacts/!(build-info)/**/*[^dbg].json'",
```

Comment:
any reason to keep supporting ethers-v4 typings?

Reply:
I think this was kept for backward compatibility, if no objections lets remove v4 typings and only support v5